### PR TITLE
Convert @itwin/hypermodeling-frontend into an extension

### PR DESCRIPTION
**Pull request by Jake-Screen**
Converting @itwin/hypermodeling-frontend into an extension by using @itwin/core-extension

### CODE REVIEW

**Comment by Jake-Screen** [comment]
Missing items from core-extension:
IModelApp.formatElementToolTip
IModelApp.uiAdmin

core-common:
Placement2d
Placement3d

core-frontend:
tryImageElementFromUrl

File: core/hypermodeling/package.json [hunk 1]
```diff
@@ -78,4 +75,4 @@
     ],
     "extends": "plugin:@itwin/itwinjs-recommended"
```
Comment (aruniverse) [inline core/hypermodeling/package.json:76]: mind adding those rules for no-duplicate imports and type assertions like we do in VCR

File: core/hypermodeling/src/SectionMarkerHandler.ts [hunk 2]
```diff
@@ -7,7 +7,7 @@
  */
 
 import { IModelApp } from "@itwin/core-frontend";
-import type { AbstractToolbarProps } from "@itwin/appui-abstract";
+import type { AbstractToolbarProps } from "@itwin/core-extension";
```
Comment (aruniverse) [inline core/hypermodeling/src/SectionMarkerHandler.ts:10]: this def does not exist in core-extension; we will need to figure out what this api looks like

File: core/hypermodeling/src/SectionMarkers.ts [hunk 3]
```diff
@@ -8,10 +8,9 @@
 
 import { assert, BeEvent, Id64String } from "@itwin/core-bentley";
 import { Point2d, Point3d, XAndY, XYAndZ } from "@itwin/core-geometry";
-import { IModelReadRpcInterface } from "@itwin/core-common";
 import {
-  BeButton, BeButtonEvent, Cluster, DecorateContext, IModelApp, InputSource, Marker, MarkerImage, MarkerSet, ScreenViewport, ViewClipTool,
-} from "@itwin/core-frontend";
+  BeButton, BeButtonEvent, Cluster, DecorateContext, InputSource, Marker, MarkerImage, MarkerSet, ScreenViewport, ViewClipTool,
```
Comment (aruniverse) [inline core/hypermodeling/src/SectionMarkers.ts:12]: did you just rename core-frontend import to core-extension, this shouldnt even compile

Comment (Jake-Screen) [reply core/hypermodeling/src/SectionMarkers.ts:12]: yes, doesnt build/compile yet, was the first step to figuring out what was missing from core-extension

File: core/hypermodeling/package.json [hunk 4]
```diff
@@ -67,7 +64,7 @@
     "webpack": "^5.76.0"
   },
   "dependencies": {
-    "@itwin/appui-abstract": "workspace:*"
+    "@itwin/core-extension": "workspace:*"
```
[the remark this answers is not in the document]
Comment (Jake-Screen) [reply core/hypermodeling/package.json]: moved to direct

**Comment by aruniverse** [comment]
We will circle back to this pr later

**Pull request closed without merging**